### PR TITLE
New package: gthumb-3.4.0

### PR DESCRIPTION
**Pull request by enguerrand**

### CODE REVIEW

File: srcpkgs/gthumb/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template build file for 'GThumb'.
+pkgname=gthumb
+version=3.4.0
+revision=1
+wrksrc="${pkgname}-${version}"
```
Comment (xtraeme) [inline srcpkgs/gthumb/template]: This `wrksrc` is redundant, this is what it's set to by default.

File: srcpkgs/gthumb/template [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+build_style=gnu-configure
+makedepends="glib-devel pkg-config intltool itstool gnome-desktop-devel"
```
Comment (xtraeme) [inline srcpkgs/gthumb/template]: Change this `makedepends` to:

```
hostmakedepends="pkg-config intltool itstool glib-devel"
makedepends="gnome-desktop-devel"
```

File: srcpkgs/gthumb/template [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+short_desc='An image viewer and browser for the GNOME Desktop'
+maintainer="Enguerrand de Rochefort <voidlinux@rochefort.de>"
+homepage="http://live.gnome.org/gthumb"
+license="GPL-3"
+distfiles="$GNOME_SITE/${pkgname}/3.4/${pkgname}-${version}.tar.xz"
```
Comment (xtraeme) [inline srcpkgs/gthumb/template]: Change `3.4` to `${version%.*}`.

**Comment by enguerrand** [comment]
Updated as per your comments. (Sorry I could have figured out some of this myself)

**Comment by xtraeme** [comment]
- This needs a `-dev` subpackage with headers, static libs and pkg-config files.
- This is missing additional dependencies for more features: exiv2, clutter, etc.

**Comment by dominikh** [comment]
@enguerrand Please address xtraeme's last comment if you're still interested in adding this package. Otherwise I'll have to close the PR due to lack of progress.

**Comment by enguerrand** [comment]
Hi Dominik

I apologize for the slow progress. I have been working on this, but at some point I got stuck and was reluctant to reach out to you guys for help because I felt I had not come very far. 
I have given it another try today and I now managed to get the optional features to build. However, the resulting package won't install with a message saying
"ERROR: gthumb-3.4.0_1: SHA256 hash is not valid: Numerical result out of range"
so this is still something I need to debug. Any pointers are welcome.

Anyway. I updated my PR with what I currently have. You'll find that the dev-subpackage is also still missing and I do not expect to solve this quickly. 

Long story short: I am still interested in submitting gthumb but progress will continue to be slow due to my lack of knowledge.

If under these circumstances you prefer to simply close the PR that is fine with me... I guess I can always just open a new one if I manage to figure this out later down the road.

**Comment by enguerrand** [comment]
Please discount the comment on the package not installing due to a checksum error. I don't know what it was but I was able to get rid of this problem by setting up void-packages from scratch again. I must have messed it up somehow. 
I'll now look into the development subpackage...

**Comment by enguerrand** [comment]
Ok. So here is what I have. Let me know what you think...

File: srcpkgs/gthumb/template [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+desc_option_gnomekeyring="Enable use of the gnome keyring"
+desc_option_brasero="Enable burning discs"
+build_options_default="exiv tiff clutter gstreamer gnomekeyring brasero"
+
+gthumb-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/lib/gthumb/extensions/*.a
+		vmove usr/lib/gthumb/extensions/*.so
```
Comment (dominikh) [inline srcpkgs/gthumb/template]: I suspect that extensions/*.so should be in the main package and that gthumb will need those to actually load the extensions.

Files that still need to be moved from the main package to -devel are:

- usr/include/

File: srcpkgs/gthumb/template [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+maintainer="Enguerrand de Rochefort <voidlinux@rochefort.de>"
+homepage="http://live.gnome.org/gthumb"
+license="GPL-3"
+distfiles="$GNOME_SITE/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
+checksum=abd1afe457c65289af31c6db7037452faa3838356480857d8cdc19f0ecde4933
+#
+## Package build options
+build_options="exiv tiff clutter gstreamer soup gnomekeyring brasero"
+desc_option_exiv="Enable support for image meta data manipulation"
+desc_option_tiff="Enable support for tiff"
```
Comment (dominikh) [inline srcpkgs/gthumb/template]: desc_option_tiff and desc_option_gstreamer already exist in common/options.descriptions – as such, you don't need to include these two in the template.

File: srcpkgs/gthumb/template [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# Template build file for 'GThumb'.
+pkgname=gthumb
+version=3.4.0
+revision=1
+build_style=gnu-configure
+LDFLAGS+=" -fPIC "
+configure_args="$(vopt_with exiv) $(vopt_with tiff) $(vopt_with clutter) $(vopt_with gstreamer)  $(vopt_with soup)  $(vopt_with gnomekeyring)  $(vopt_with brasero)"
```
Comment (dominikh) [inline srcpkgs/gthumb/template]: Please wrap long lines. Consider 80-90 characters per line the soft limit.

**Comment by dominikh** [comment]
It's looking fine overall. I commented on the parts that need improvement. I'll defer to @xtraeme for a final look over the template, but you're getting close.

**Comment by enguerrand** [comment]
Hi Dominik,

thanks a lot for taking the time to review again and for the helpful comments, it is much appreciated. 
I updated the PR accordingly...
